training-maquetado/scss/application.scss

```diff
@@ -1,2 +1,6 @@
 @import '_equalizer';
-@import '_styles';
+@import '_titles';
+
+* {
```

Comment:
i would put this in `styles.scss`

Reply:
current `_styles.scss` or new file `_sizes.scss`?